src/pudl/metadata/resources/ferc1.py

```diff
@@ -537,6 +537,40 @@
         "etl_group": "ferc1",
         "field_namespace": "ferc1",
     },
+    "retained_earnings_ferc1": {
+        "description": "Reatained Earnings - The structed part of schedule 118.",
+        "schema": {
+            "fields": [
+                "utility_id_ferc1",
+                "report_year",
+                "record_id",
+                "earnings_type",
+                "amount",
+                "starting_balance",
+                "ending_balance",
+                "balance",
+                "ferc_account",
+                "row_type_xbrl",
+            ],
+            "primary_key": [
+                "utility_id_ferc1",
+                "report_year",
+                "earnings_type",
+            ],
+        },
+        "sources": ["ferc1"],
+        "etl_group": "ferc1",
+        "field_namespace": "ferc1",
+    },
+    "retrained_earnings_appropriations_ferc1": {
+        "description": "Reatained Earnings - some of the structed part of schedule 118.",
```

Comment:
ReAtained earnings also misspelled here.

Reply:
when you find these things, you can just say it once in the pr.